src/commonMain/kotlin/assertk/assertions/support/support.kt

```diff
@@ -86,3 +88,14 @@ private fun formatName(name: String?): String {
         " [$name]"
     }
 }
+
+private val specialWhitespace = Regex("[\r\n\t]|  +")
```

Comment:
Any specific reason for not using `\s`?

Reply:
Lack of familiarity with regex really.

Reply:
Actually, the space stuff in that regex is left over from when I was trying to test if they should be revealed (My original thought was to only reveal if the diff looked the same otherwise, ended up with just always showing them in the diff). Changing the pattern to just the chars that are being replaced.